drivers/oled/glcdfont_16x12.c

```diff
@@ -0,0 +1,35 @@
+#include "progmem.h"
```

Comment:
Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Reply:
Thank you for your patience.
Added.

I only resized it, so I should probably list it with the "drivers\oled\glcdfont.c" author, but since the "drivers\oled\glcdfont.c" did not include a license header, I have only included my account.